core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/WorkspaceApiServiceSpec.scala

```diff
@@ -1430,6 +1430,47 @@ class WorkspaceApiServiceSpec extends ApiServiceSpec {
       }
   }
 
+  // Send Change Notification for Workspace require WRITE access. Accept if OWNER or WRITE; Reject if READ or NO ACCESS
```

Comment:
Also, look in those tests to see how they check that a notification was actually sent.